packages/salesforcedx-vscode-core/test/vscode-integration/commands/templates/customTemplates.test.ts

```diff
@@ -66,12 +66,12 @@ describe('Custom Templates Create', () => {
     openTextDocumentStub.restore();
     sendCommandEventStub.restore();
     sendExceptionStub.restore();
-    getConfigValue.restore();
+    getTemplatesDirectoryStub.restore();
```

Comment:
nit: should we add a sandbox here so we don't have to restore all the things and have the ability to parallelize the tests?